integration/root_integration_test.go

```diff
@@ -0,0 +1,36 @@
+//+build test_as_root
+
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"os"
+	"testing"
+
+	"gopkg.in/check.v1"
+)
+
+func TestIntegrationsRoot(t *testing.T) { check.TestingT(t) }
```

Comment:
this is only required for packages using [gocheck](https://github.com/go-check/check) (which we are moving away from) to hook into testing subsystem - unnecessary here.

Reply:
this was marked as resolved but the function isn't removed
i think this whole file can be removed